src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -1003,10 +1006,53 @@ class ReservedDocumentViewSet(viewsets.ViewSet):
             "binnen de bronorganisatie en het documentnummer kan daarna niet hergebruikt worden"
         ),
         request=ReservedDocumentSerializer,
-        responses={status.HTTP_201_CREATED: ReservedDocumentSerializer},
+        responses={
+            status.HTTP_201_CREATED: OpenApiResponse(
+                examples=[
+                    OpenApiExample(
+                        "Enkele reservering",
+                        value={"identificatie": "DOCUMENT-2025-0000000001"},
+                        response_only=True,
+                    ),
+                    OpenApiExample(
+                        "Meerdere reserveringen",
+                        value=[
+                            {"identificatie": "DOCUMENT-2025-0000000001"},
+                            {"identificatie": "DOCUMENT-2025-0000000002"},
+                        ],
+                        response_only=True,
+                    ),
+                ],
+            )
+        },
```

Comment:
```suggestion
        examples=[
            OpenApiExample(
                "Enkele reservering",
                value={"identificatie": "DOCUMENT-2025-0000000001"},
                response_only=True,
            ),
            OpenApiExample(
                "Meerdere reserveringen",
                value=[
                    {"identificatie": "DOCUMENT-2025-0000000001"},
                    {"identificatie": "DOCUMENT-2025-0000000002"},
                ],
                response_only=True,
            ),
        ],
```

the examples don't seem to work, by using examples directly it seems it does and this way the error examples are also added.